crm_lead_to_event_registration/wizard/crm_lead_event_pick.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class WizardModel(models.TransientModel):
+    _name = "crm.lead.event.pick"
+
+    lead_id = fields.Many2one(
+        "crm.lead",
```

Comment:
Could you use explicit keyword parameters, as defined in guidelines? https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field

```
    lead_id = fields.Many2one(
        comodel_name="crm.lead",
        string="Lead/Opportunity",
(...)
```